src/adapters/api-blueprint-adapter.coffee

```diff
@@ -67,6 +67,48 @@ ensureObjectOfObjects = (data, key = 'name') ->
     data
 
 
+# # Merges multiple headers with the same key into one
+# This is needed in A1 -> legacy header merge / transformation,
+# since A1 can contain multiple values for given
+# header key, legacy cannot
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde' }
+#     { name: 'Set-Cookie', value: 'Alan Turing' }
+#   ]
+#
+# is turned into
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde; Alan Turing' }
```

Comment:
Right, thanks. Reflected in https://github.com/apiaryio/metamorphoses/pull/82/commits/64b29b68e66fcc98d56fe14aa8944e4b4475359a.